### PR TITLE
initial checkin of a llama3.1 example

### PR DESCRIPTION
**Pull request by amygdala**

### CODE REVIEW

File: ml_examples/llama31/overview.md
Comment (wnojopra) [inline ml_examples/llama31/overview.md:2]: This file seems out of place. Was it accidently included? Or perhaps it should be moved to a different directory?

File: ml_examples/llama31/README.md
Comment (wnojopra) [inline ml_examples/llama31/README.md:10]: Do you know what permissions are required for the token? For example, if a read-only token is sufficient, that would be good to mention.

File: ml_examples/llama31/overview.md
Comment (wnojopra) [reply ml_examples/llama31/overview.md:2]: OHH I see now its part of the notebook.

Maybe just add this line to the top?
```suggestion
==NOTE - this file is intended to be used as input for the vwb_8b_v100_llama31_hf.ipynb notebook==

## What is Verily Workbench?
```

**Review by wnojopra — APPROVED** [review]
Overall LGTM. I'm waiting for approval to access the model so I couldn't execute most of the code. I did however ran the code up to that point, read all the markdown and tested all the links.

Just two comments inline.

File: ml_examples/llama31/overview.md
Comment (amygdala) [reply ml_examples/llama31/overview.md:2]: Thanks, good catch.  haha, it didn't occur to me that this would be confusing :).  I'll add a note to the top as you suggest.

File: ml_examples/llama31/README.md
Comment (amygdala) [reply ml_examples/llama31/README.md:10]: Yes, read-only is sufficient, I will add that.

**Comment by amygdala** [comment]
thanks @wnojopra !!

**Pull request merged**
